docs/release-notes/v0.38.0.md

```diff
@@ -0,0 +1,74 @@
+# Announcing Radius v0.38.0
+
+Today we're happy to announce the release of Radius v0.38.0. Check out the [highlights](#highlights) below, along with the [full changelog](#full-changelog) for more details.
+
+We would like to extend our thanks to all the [new](#new-contributors) and existing contributors who helped make this release possible!
+
+## Intro to Radius
+
+If you're new to Radius, check out our website, [radapp.io](https://radapp.io), for more information. Also visit our [getting started guide](https://docs.radapp.io/getting-started/) to learn how to install Radius and create your first app.
+
+## Highlights
+
```

Comment:
Note that the documentation for "Secret store reference in Dapr components" is still [pending](https://github.com/radius-project/radius/pull/7823#issuecomment-2344366146), but the overall link here to the Dapr schema should remain the same.

```suggestion

### Secret stores in environment variables
Secret stores can now be referenced in the environment variables of a container resource. Previously, the only way to provide secrets to a container resource in Radius is to mount them through a secret volume, so the experience is now much simpler for users who only require the secrets to be passed into the container via environment variables. See the updated [container resource format](https://docs.radapp.io/reference/resource-schema/core-schema/container-schema/) for more information. Many thanks to community contributor @superbeeny for adding this functionality!

### Secret store reference in Dapr components
Support has been added for building Dapr components with an optional reference to a Dapr secret store. This means that users may now reference non-Kubernetes secrets within the Dapr resources, closing a feature gap between Radius and the Dapr `spec.metadata` specification. See the [Radius resource schema](https://docs.radapp.io/reference/resource-schema/dapr-schema/) and [Dapr component secrets reference guide](https://docs.dapr.io/operations/components/component-secrets/) for more information. Many thanks to community contributor @SoTrx for adding this functionality!

### Bicep private registry support 
Support is now available for authentication into private registries for Bicep Recipes using basic auth, Azure workload identity, and AWS IRSA. See the how-to guide on [private Bicep registries](https://edge.docs.radapp.io/guides/recipes/howto-private-bicep-registry/) to learn more.
```

Reply:
There is a how to guide contribution from @SoTrx on [how to reference Dapr secret stores](https://github.com/radius-project/docs/pull/1234) which needs to be reviewed. Once reviewed and merged, we can attach the link here

Reply:
Edge docs are referenced under Bicep private registry. Needs to be updated  to https://docs.radapp.io/guides/recipes/howto-private-bicep-registry/

Reply:
Thanks @Reshrahim, updated both.